src/tests/ftest/nvme/pool_extend.py

```diff
@@ -53,9 +53,9 @@ def run_nvme_pool_extend(self, num_pool, oclass=None):
 
         # Create the pools
         pools = []
-        for _ in range(0, num_pool):
-            pools.append(self.get_pool(properties="reclaim:disabled"))
-            # pools[-1].set_property("reclaim", "disabled")
+        for index in range(0, num_pool):
+            pools.append(self.get_pool(namespace="/run/pool_qty_{}/*".format(num_pool),
+                         properties="reclaim:disabled"))
```

Comment:
Did you mean to do this?
```suggestion
        for index in range(0, num_pool):
            pools.append(self.get_pool(namespace="/run/pool_qty_{}/*".format(index),
                         properties="reclaim:disabled"))
```

Reply:
No... I wanted to use format(num_pool) here..

When we create 2 pools, we want to make sure each pool uses pool_qty_{2} : size.

Reply:
Then, curious why change `for _` -> `for index`? Since `index` is unused?

Reply:
Yeah, right... I can do that....

Reply:
Done.

Reply:
Sorry and thanks. I thought I was misunderstanding something :)